.github/workflows/regression.yml

```diff
@@ -0,0 +1,67 @@
+name: regression
+
+on:
+  push:
+    branches: [main]  # Adjust this if your default branch has a different name
+  pull_request:
+    types: [labeled]
+  workflow_dispatch:  # Add manual trigger capability
+
+jobs:
+  regression:
+    runs-on: blacksmith-4vcpu-ubuntu-2204
+    steps:
+    # Untrusted action -> specific githash
+    - uses: snnaplab/get-labels-action@f426df40304808ace3b5282d4f036515f7609576
```

Comment:
Maybe we should make this an action ourselves. Perhaps good publicity, not too complicated and fuck it's a PITA right now